web/source/kmwosk.js

```diff
@@ -791,12 +791,14 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
 
       // Test for fall back to U_xxxx key id - For this first test, we ignore the key code
       // and use the keyName
-      if((keyName.substr(0,2) == 'U_')) { 
-        ch=String.fromCharCode(parseInt(keyName.substr(2,6),16));
-        if(ch <= 32 || (ch > 127 && ch < 160)) {
-          // 127 - 160 refer to Unicode control codes.
+      if((keyName.substr(0,2) == 'U_')) {
+        var codePoint = parseInt(keyName.substr(2,6), 16);
+        ch=String.fromCharCode(codePoint);
+        if((0x0 <= codePoint && codePoint <= 0x1F) || (0x80 <= codePoint) && (codePoint <= 0x9F)) {
```

Comment:
If I remember correctly, you mentioned that doing the comparison with the final characters had issues that could be avoided by checking the code points instead?  What was the issue?  Maybe add a little documentation to why we need this done on the code point specifically?

Reply:
The previous comparison ended up processing nbsp (U_00A0) as a Unicode control code, so that character wasn't getting generated. I've updated the comments to distinguish codePoint (number) from ch (string).